### ISSUE TITLE
Swipe with multimenues

### ISSUE THREAD

**Issue by thawkins** [artberri/sidr#290]
At the moment if you have two menues, one on each side,  if you swipe left the left menu opens,  right now i have the option to have swipe right open the right menu (which closes left), or just close left if it is open. 

What i would like to do is close left if it is open and then exit, so that a second right swipe would then open the right menu. Alternativly is there a simple way of testing if a menu is slready open or not, so i can make the swipe/click action conditional on which menu is already open if any.

**Comment by artberri** [comment]
@thawkins Right now you can now it checking the html element's classes.

If you have Sidr menu which name is `foo`, you can check if it's already opened with:
`$('html').hasClass('foo-open');`

But yeah, you've catched a good point. So, in the following versions I'll try to give you an API to check if the menu is opened, or which menu is currently opened.

Thank you. I'll keep this issue opened until I implement that.

**Comment by artberri** [comment]
Finally I've released a new version (2.2.0) that implements the ability to get the status object where you can get that information:

```
var status = $.sidr('status');
```

The status content will be:

```
{
    moving: (boolean), // A boolean indicating if the menu is currently being moved
    opened: (string || false) // Name of the opened menu or false if all are closed
}
```
